lib/stdlib/test/ets_SUITE.erl

```diff
@@ -2568,6 +2568,8 @@ update_element_do(Tab,Tuple,Key,UpdPos) ->
                       %%io:format("update_element(~p)~n",[PosValArg]),
                       ArgHash = erlang:phash2({Tab,Key,PosValArg}),
                       true = ets:update_element(Tab, Key, PosValArg),
+		      [DefaultObj] = ets:lookup(Tab, Key),
+		      true = ets:update_element(Tab, make_ref(), PosValArg, DefaultObj),
```

Comment:
It's a bit thin with only one positive test call to update_element/4 that does not check the result.

Reply:
~@juhlig~ We (:grin:) wanted to see if this addition would be accepted first, before writing more elaborate tests.

Reply:
I'll see to it tomorrow.

Reply:
@sverker I augmented the tests a bit in the last commit, is that enough or do you want more?

Reply:
Well, at least one call when the default is not used would be nice. But maybe we don't need to put that in update_element_do to be repeated in absurdum.

Reply:
@sverker I added a test that checks `update_element/4` with non-existing and existing object. TBH, I had some trouble with trying to wedge it into the existing test, but in the end settled on a separate case.